Build issues : multiple definition of 'attack_count'

Hi,
I've a little compiling problem, here is the error:

```
gcc -g -W -Wall -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=mdk4-v1 -DCONFIG_LIBNL30 -DCONFIG_LIBNL -I/usr/include/libnl3  -g -O3 -Wall -Wextra   mdk4.c osdep/libosdep.a debug.o helpers.o mac_addr.o linkedlist.o greylist.o dumpfile.o packet.o brute.o osdep.o channelhopper.o ghosting.o fragmenting.o attacks/attacks.o attacks/auth_dos.o attacks/beacon_flood.o attacks/countermeasures.o attacks/deauth.o attacks/dummy.o attacks/eapol.o attacks/fuzzer.o attacks/ieee80211s.o attacks/probing.o attacks/wids.o -o mdk4 -lnl-genl-3 -lnl-3  -lm -Losdep -losdep -lpthread -lpcap
/usr/bin/ld: channelhopper.o:/home/jordan/mdk4/src/./attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cczBuTgT.o:/home/jordan/mdk4/src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/attacks.o:/home/jordan/mdk4/src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cczBuTgT.o:/home/jordan/mdk4/src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/auth_dos.o:/home/jordan/mdk4/src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cczBuTgT.o:/home/jordan/mdk4/src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/beacon_flood.o:/home/jordan/mdk4/src/attacks/beacon_flood.c:34: multiple definition of `bssid'; attacks/auth_dos.o:/home/jordan/mdk4/src/attacks/auth_dos.c:50: first defined here
/usr/bin/ld: attacks/beacon_flood.o:/home/jordan/mdk4/src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cczBuTgT.o:/home/jordan/mdk4/src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/countermeasures.o:/home/jordan/mdk4/src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cczBuTgT.o:/home/jordan/mdk4/src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/deauth.o:/home/jordan/mdk4/src/attacks/deauth.c:19: multiple definition of `bssid'; attacks/auth_dos.o:/home/jordan/mdk4/src/attacks/auth_dos.c:50: first defined here
/usr/bin/ld: attacks/deauth.o:/home/jordan/mdk4/src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cczBuTgT.o:/home/jordan/mdk4/src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/dummy.o:/home/jordan/mdk4/src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cczBuTgT.o:/home/jordan/mdk4/src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/eapol.o:/home/jordan/mdk4/src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cczBuTgT.o:/home/jordan/mdk4/src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/fuzzer.o:/home/jordan/mdk4/src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cczBuTgT.o:/home/jordan/mdk4/src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/ieee80211s.o:/home/jordan/mdk4/src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cczBuTgT.o:/home/jordan/mdk4/src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/probing.o:/home/jordan/mdk4/src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cczBuTgT.o:/home/jordan/mdk4/src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/wids.o:/home/jordan/mdk4/src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cczBuTgT.o:/home/jordan/mdk4/src/attacks/attacks.h:39: first defined here
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:36: mdk4] Error 1
make[1]: Leaving directory '/home/jordan/mdk4/src'
make: *** [Makefile:10: all] Error 2
```

Thanks in advance for your answer !

--- a/README.md
+++ b/README.md
@@ -20,6 +20,7 @@ MDK4 is licenced under the GPLv2 or later.
 		cd mdk4
 		make
 		sudo make install
+		# Using Arch Linux (and derived) append: CC=clang
 
 
 # Features

--- a/README.md
+++ b/README.md
@@ -20,7 +20,7 @@ MDK4 is licenced under the GPLv2 or later.
 		cd mdk4
 		make
 		sudo make install
-		# Using Arch Linux (and derived) append: CC=clang
+		# Using Arch Linux (and derived) append `CC=clang` after any `make` in commands. 
 
 
 # Features

